### ISSUE TITLE
Proposal: Make CaptureElement available for WinUI3 & Windows App SDK.

### ISSUE THREAD

**Issue by lighterhy** [microsoft/microsoft-ui-xaml#8214]
Displaying a camera preview is an essential feature in a modern software development kit.

**Comment by castorix** [comment]
> Displaying a camera preview is an essential feature in a modern software development kit.

There are several methods like _DirectShow_ or _IMFCaptureEngine_ which can be used

**Comment by lighterhy** [comment]
> > Displaying a camera preview is an essential feature in a modern software development kit.
> 
> There are several methods like _DirectShow_ or _IMFCaptureEngine_ which can be used

It takes a lot of work when using DirectShow or Media Foundation directly in C#. Especially if you want to improve performance with DXVA and EVR.
It's now 2023, and it's an essential foundational feature in modern apps.

**Comment by castorix** [comment]
> It takes a lot of work when using DirectShow or Media Foundation directly in C#. Especially if you want to improve performance with DXVA and EVR. It's now 2023, and it's an essential foundational feature in modern apps.

Yes, but IMFCaptureEngine is not too difficult to use (I just translated the MS SDK C++ sample into C#) and the result can be displayed inside an XAML Image control

**Comment by castorix** [comment]
I just uploaded a test with _IMFCaptureEngine_ : [WinUI3_MFCaptureEngine](https://github.com/castorix/WinUI3_MFCaptureEngine/tree/master)
But as I said in comments, I sometimes got a crash; maybe if someone can reproduce it and find the reason...
(only tested on Windows 10)
